_topics/en/compact-block-relay.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Compact block relay
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BIP152
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Bandwidth Reduction
+  - Mining
+  - P2P Network Protocol
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: BIP152
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: Bitcoin Core PR Review Club about dropping support for v1 compact blocks
+    url: /en/newsletters/2021/02/10/#bitcoin-core-pr-review-club
+
+  - title: "Bitcoin Core #20764 adds BIP152 high bandwidth indicator to bitcoin-cli"
+    url: /en/newsletters/2021/02/10/#bitcoin-core-20764
+
+  - title: "Bitcoin Core #19776 updates `getpeerinfo` RPC with BIP152 peer status"
+    url: /en/newsletters/2020/12/16/#bitcoin-core-19776
+
+  - title: Consequences of not backporting taproot relay on compact block efficiency
+    url: /en/newsletters/2020/07/29/#dont-relay-taproot
+
+## Optional.  Same format as "primary_sources" above
+# see_also:
+#   - title:
+#     link:
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Compact block relay** is a protocol that allow two nodes with
+  roughly similar sets of unconfirmed transactions to minimize both the
+  bandwidth and the latency required to transfer a block that confirms
+  many of those same transactions.
+
+---
+
+By default, most full nodes are configured to receive relayed
+unconfirmed transactions, which the nodes store in their mempools and
+further relay to their other peers.  Ultimately those transactions
+reach miners, who attempt to include them in a block.  When that new
+block is itself relayed back to nodes, it typically consists almost
+entirely of transactions that the nodes already have in their mempools.
+
+Relaying those transactions a second time is redundant, and it's that
+redundancy that [BIP152][] compact blocks aims to eliminate.  A sending
+node constructs a compact block by replacing each transaction it
+believes its peer has with a 6-byte hash of that transaction's witness
+txid (wtxid).  This can take a multimegabyte block with thousands of
```

Comment:
You only mention the hash of the wtxid here, but right at the end of the document, reveal that it could also be the hash of the txid. Perhaps introduce that here.